### ISSUE TITLE
RTL-support for MAUI

### ISSUE THREAD

**Issue by grabnerM** [UXDivers/Grial-UI-Kit-Support#865]
**Is your feature request related to a problem? Please describe.**
RTL-support does not work in MAUI yet. Since the xamarin end of life date is coming closer, we have to ship our app to MAUI. Unfortunately there is not RTL-support which is crucial for our app since we are supporting "arabic"

**Describe the solution you'd like**
A working RTL-support for MAUI

**Describe alternatives you've considered**
Switching to an different framework which supports RTL

**Comment by hksharafudheen** [comment]
@grabnerM Did you get a solution for this?

**Comment by grabnerM** [comment]
@hksharafudheen Yeah, we did. In MAUI there is the Property FlowDirection. If you set this to "RightToLeft" the app switches the design on its own. Now you can add this to every ContentPage. To prevent missing this on one page, we implemented a custom ContentPage which derives the standard ContentPage. This custom ContentPage is just an C#-class. So now every page we use is our custom page. 

For this custom page we set a styling in our main resources-file.
`<Setter Property="FlowDirection" Value="{ DynamicResource PageFlowDirection }" />`
The value "PageFlowDirection" gets set in our LTR- and RTL-Resources-File. Obviously in LTR its 'LeftToRight' and in RTL its 'RightToLeft'. If we set the language to arabic the used resource-file gets changed to the RTL-resource-file and the FlowDirection changes
